CalculatorApp.Tests/AddMethodShould.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using CalculatorApp;
+using FluentAssertions;
+namespace CalculatorApp.Tests
+{
+    public class AddMethodShould
+    {
+        private Calculator _sut;
+        public AddMethodShould()
+        {
+            _sut = new Calculator();
+        }
+
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData(" ")]
+        public void Return_Zero_When_Empty_Or_Null_String(string numbers)
```

Comment:
test method name should be : MethodUnderTestName_Should_When

so, this method should have this name
Add_ShouldReturnZero_WhenEmptyOrNullString 

Apply to all methods plz